src/psbt.cpp

```diff
@@ -69,6 +69,9 @@ bool PartiallySignedTransaction::GetInputUTXO(CTxOut& utxo, int input_index) con
     PSBTInput input = inputs[input_index];
     int prevout_index = tx->vin[input_index].prevout.n;
     if (input.non_witness_utxo) {
+        if (prevout_index >= input.non_witness_utxo->vout.size()) {
```

Comment:
I'm afraid this will trigger a sign-compare warning when compiling with Clang due to LHS being `int` and the RHS being `unsigned long` (`size_t`).

Reply:
Fixed